packages/rrweb/src/record/observers/canvas/canvas-manager.ts

```diff
@@ -227,16 +375,19 @@ export class CanvasManager implements CanvasManagerInterface {
         lastSnapshotTime &&
         timestamp - lastSnapshotTime < timeBetweenSnapshots
       ) {
-        rafId = onRequestAnimationFrame(takeCanvasSnapshots);
+        rafId = requestAnimationFrame(takeCanvasSnapshots);
```

Comment:
I think `onRequestAnimationFrame` is right (Francesco recently made this change).